client/my-sites/sharing/buttons/buttons.jsx

```diff
@@ -96,9 +96,9 @@ class SharingButtons extends Component {
 
 		return (
 			<form onSubmit={ this.saveChanges } id="sharing-buttons" className="sharing-settings sharing-buttons">
-				<QuerySiteSettings siteId={ siteId } />
-				<QueryPostTypes siteId={ siteId } />
-				<QuerySharingButtons siteId={ siteId } />
+				{ siteId && <QuerySiteSettings siteId={ siteId } /> }
+				{ siteId && <QueryPostTypes siteId={ siteId } /> }
+				{ siteId && <QuerySharingButtons siteId={ siteId } /> }
```

Comment:
Appears `<QuerySiteSettings />` and `<QuerySharingButtons />` are smart enough to ignore a falsey `siteId`, but probably good to be safe as you've done here regardless.